ixwebsocket/IXHttpServer.cpp

```diff
@@ -81,27 +67,23 @@ namespace ix
     void HttpServer::handleConnection(std::unique_ptr<Socket> socket,
                                       std::shared_ptr<ConnectionState> connectionState)
     {
-        _connectedClientsCount++;
-
         auto ret = Http::parseRequest(socket, _timeoutSecs);
         // FIXME: handle errors in parseRequest
 
         if (std::get<0>(ret))
         {
-            auto response = _onConnectionCallback(std::get<2>(ret), connectionState);
-            if (!Http::sendResponse(response, socket))
-            {
-                logError("Cannot send response");
+            auto request = std::get<2>(ret);
+            std::shared_ptr<ix::HttpResponse> response;
```

Comment:
Can you run the formater (clang-format) ?
{ braces go on their own lines etc...)

Reply:
Manually indenting is fine too, as sometimes running the formatter modifies a lot of code.